docs/merged_objects.md

```diff
@@ -192,7 +192,9 @@ Each such list will contain the following fields:
 | `cellassign_reference_version` | If cell typing with `CellAssign` was performed and completed successfully, the version of the reference dataset source. For references obtained from `PanglaoDB`, the version scheme is a date in ISO8601 format                                                                                                                                                                   |
 | `scimilarity_model` | If cell typing with `SCimilarity` was performed, the name of the foundation model used to run `SCimilarity` | 
 | `consensus_celltype_methods` | If consensus cell types are present, a vector with the names of the automated methods used to generate consensus cell type annotations |
-| `infercnv_reference_celltypes` | Vector of consensus cell type labels which would be specified for the normal reference cells in `inferCNV` reference. Only present when consensus cell types were assigned, regardless of whether CNV inference is performed                                                                                                                                                       |
+| `infercnv_reference_celltypes` | Vector of consensus cell type labels which would be specified for the normal reference cells in `inferCNV` reference. Only present when consensus cell types were assigned and CNV inference was attempted. If CNV inference was attempted and the sample is obtained from normal or non-cancerous tissue this will not be present                                                                                                                                                   |
+| `infercnv_num_reference_cells` | The total number of normal reference cells included in the `inferCNV` reference. Only present when consensus cell types were assigned and CNV inference was attempted. If CNV inference was attempted and the sample is obtained from normal or non-cancerous tissue this will be `NA` | 
```

Comment:
```suggestion
| `infercnv_num_reference_cells` | The total number of normal reference cells included in the `inferCNV` reference. Only present when consensus cell types were assigned and CNV inference was attempted. If CNV inference was attempted and the sample was obtained from normal or non-cancerous tissue this will be `NA` | 
```